squid/.vscode/settings.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
And this too please

Reply:
vscode settings it's about personal preferences, similar to .ide as in my case I use a Webstorm. To ensure the same formatting styles we have a client configured. Do you know if there are any use cases I am missing to keep the vscode settings?

Reply:
I thought it was a bad idea to commit this indeed, but I've been struggling with this setting personally and wanted to make sure not to receive PRs with 100s of indentation changes. I've added an eslint rule subsequently, this is then probably not needed any more.